selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/theia/TheiaEditor.java

```diff
@@ -96,18 +102,50 @@ public void performPasteAction() {
   }
 
   public List<String> getEditorText() {
-    List<WebElement> editorLinesElements =
-        seleniumWebDriverHelper.waitVisibilityOfAllElements(By.xpath(EDITOR_LINE_XPATH));
-
-    editorLinesElements.sort(
-        (firstElement, secondElement) ->
-            getEditorLinePixelCoordinate(firstElement)
-                - getEditorLinePixelCoordinate(secondElement));
-
-    return editorLinesElements
-        .stream()
-        .map(element -> seleniumWebDriverHelper.waitVisibilityAndGetText(element))
-        .collect(Collectors.toList());
+    // In this realization each line element will be found exactly before using
+    // This realization allows as to avoid "StaleElementReferenceException"
+    final int editorLinesCount =
+        seleniumWebDriverHelper.waitVisibilityOfAllElements(By.xpath(EDITOR_LINE_XPATH)).size();
+
+    List<Integer> linesCoordinates = getEditorLinesPixelCoordinates(editorLinesCount);
+
+    linesCoordinates.sort((first, second) -> first - second);
```

Comment:
Could you, please, comment on why we need to sort coordinates?

Reply:
Also, it looks reasonable to return has already sorted coordinates from the `getEditorLinesPixelCoordinates()` method.

Reply:
Another point - does `coordinate` mean `line number`? If so, `line number` sounds more natural, IMHO.